### PR TITLE
issue #45 - excel integration

### PR DESCRIPTION
**Pull request by zeala**

### CODE REVIEW

File: src/client/src/ui/blotter/model/blotterModel.js [hunk 1]
```diff
@@ -41,6 +41,8 @@ export default class BlotterModel extends ModelBase {
     _log.info(`Blotter starting`);
     this._subscribeToConnectionStatus();
     this._regionManagerHelper.addToRegion();
+
+    setTimeout(this.subscribeToOpenFinEvents.bind(this), 1000);
```
Comment (zeala) [inline src/client/src/ui/blotter/model/blotterModel.js]: @KeithWoods do we have an event that is dispatched when the DOM is ready? OpenFin has an internal listener for this even when it sets up the desktop API. the 'init' event is dispatched too early, so I can't use that one.

**Comment by fdore** [comment]
As a general comment, I don't think it is the models' responsibility to integrate with openfin. I would rather move the integration to the service layer.

File: src/client/src/ui/spotTile/model/spotTileModel.js [hunk 2]
```diff
@@ -161,8 +162,32 @@ export default class SpotTileModel extends ModelBase {
       this._log.warn(`Ignoring execute request as we can't trade with pricing and execution down`);
     }
   }
+  
+  _subscribeToClosePositionRequests(){
```
Comment (fdore) [inline src/client/src/ui/spotTile/model/spotTileModel.js]: I would move this logic to the execution service, given we don't seem to interact with the spot tile at all.

File: src/client/src/ui/blotter/model/blotterModel.js [hunk 3]
```diff
@@ -55,6 +56,13 @@ export default class BlotterModel extends ModelBase {
     this._regionManagerHelper.popout('Blotter', 850, 280);
   }
 
+  subscribeToOpenFinEvents(){
```
Comment (fdore) [inline src/client/src/ui/blotter/model/blotterModel.js]: I would move that to the blotter service

File: src/client/src/services/mappers/priceMapper.js [hunk 4]
```diff
@@ -29,6 +30,15 @@ export default class PriceMapper {
     );
   }
 
+  static mapToStopPriceDto(spotPrice:SpotPrice):Object{
```
Comment (fdore) [inline src/client/src/services/mappers/priceMapper.js]: should be mapToSpotPriceDto

**Pull request merged**